src/components/views/messages/MAudioBody.js

```diff
@@ -96,9 +96,13 @@ export default class MAudioBody extends React.Component {
 
         const contentUrl = this._getContentUrl();
 
+        const shouldAutoPlay = MatrixClientPeg.get().credentials.userId !==
+          this.props.mxEvent.sender.userId &&
+          content.autoplay;
+
         return (
             <span className="mx_MAudioBody">
-                <audio src={contentUrl} controls />
+                <audio src={contentUrl} controls autoPlay={shouldAutoPlay} />
```

Comment:
True, but I'd prefer to keep this simple and iterate on the logic rather than have this PR stuck forever (given the demand.)

What I had in mind:
- Only autoplay if the current user has also sent a PTT message in the same room (so you need to send one first to have others' messages autoplay to you)
- Play automatically when you receive a message (and the above condition is satisfied)
- Queue up newer messages as they come in, if there's something already playing

(this can't work with the autocomplete attribute though)

Also, while I'm currently reusing the microphone icon, it might be worth considering if we want to change it.